pkg/sql/logictest/testdata/logic_test/create_as

```diff
@@ -132,7 +132,7 @@ query III colnames
 SELECT * FROM bar
 ----
 a  b  c
-1  2  4
+1  2  5
```

Comment:
Why did this change?

Reply:
foo is `CREATE TABLE foo (x, y, z) AS SELECT catalog_name, schema_name, sql_path FROM information_schema.schemata`, and we added a new schema.